src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -350,3 +355,66 @@ def _update_offer_bid_trade(self) -> None:
         for area_uuid, area_result in self.flattened_area_core_stats_dict.items():
             self.bids_offers_trades[area_uuid] = {
                 k: area_result[k] for k in ("offers", "bids", "trades")}
+
+
+class CoefficientEndpointBuffer(SimulationEndpointBuffer):
+    """Calculate the endpoint results for the Coefficient based market."""
+
+    def _calculate_and_update_last_market_time_slot(self, area):
+        pass
+
+    def _populate_core_stats_and_sim_state(self, area: "Area"):
```

Comment:
Here the type is `Area`, but AFAIU this class will only be used for market simulations (not for coefficient simulations). Is this correct?

Reply:
Agreed, and fixed some other type mismatches in endpoint buffer along the way.